src/leo-client-app/src/components/SatelliteInfo.tsx

```diff
@@ -66,42 +67,53 @@ const SatelliteInfo: React.FC = () => {
     };
   }, []);
 
-  const { positionEci, velocityEci, longitude, latitude, height, azimuth, elevation, rangeSat } = state;
+  const {
```

Comment:
nit: extra space

Reply:
that's how eslint destructures an object if the line is too long

Reply:
man thats ugly (i should check what linter im using)